<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>devItem</class>
 <widget class="QWidget" name="devItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>317</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>LXGW WenKai Lite</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:0px;
border-radius:8px;
background-color: rgba(255, 255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="toolTip">
      <string>点击选择此设备</string>
     </property>
     <property name="styleSheet">
      <string notr="true">/*background-color: rgb(255, 255, 255);*/
border:0px solid #BDBDBD;border-radius:8px;background-color: rgba(0, 0, 0, 0.4);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_phone" native="true">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">/*background-color: rgb(255, 255, 255);*/
border:0px solid #BDBDBD;border-radius:8px;background-color: rgba(0, 0, 0, 0.4);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">/*background-color: rgb(255, 255, 255);*/
border:0px solid #BDBDBD;
border-radius:8px;
background-color:white;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>LXGW WenKai Lite</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(0,0,0,0.1);
border:0px solid #BDBDBD;
border-radius:15px;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_model_info">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>LXGW WenKai Lite</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(0,0,0,0.1);
border:0px solid #BDBDBD;
border-radius:15px;</string>
              </property>
              <property name="text">
               <string>型号</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_addr_info">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>LXGW WenKai Lite</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(0,0,0,0.1);
border:0px solid #BDBDBD;
border-radius:15px;</string>
              </property>
              <property name="text">
               <string>序列号</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_device_info">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>LXGW WenKai Lite</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(0,0,0,0.1);
border:0px solid #BDBDBD;
border-radius:15px;</string>
              </property>
              <property name="text">
               <string>设备代号</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_stat_info">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>30</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>LXGW WenKai Lite</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(0,0,0,0.1);
border:0px solid #BDBDBD;
border-radius:15px;</string>
              </property>
              <property name="text">
               <string>状态未知</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>8</width>
               <height>8</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>8</width>
               <height>8</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">/*background-color: rgb(255, 255, 255);*/
border:0px solid #BDBDBD;
border-radius:4px;
background-color:white;</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>10</width>
               <height>10</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
